src/ce/certcreate.py

```diff
@@ -1,8 +1,4 @@
-<<<<<<< HEAD:src/ce/certcreate.py
```

Comment:
Looks like some merge conflict items didn't get resolved here.

Reply:
I had to remove this because this problem is in master. It is resolved in my branch